### DISCUSSION TITLE
Account Blocked without any reason

### DISCUSSION THREAD

**Discussion by Zeshan-Shakil** [vercel/community#3700] [Account Issue] [answered]
### Question

this is my first time on vercel i want to make account by github but i dont know why you blocked my account

**Comment by amyegan** [comment, marked as answer]
Hey there @Zeshan-Shakil. Due to the sensitive nature of account information, we need to handle this in a support case rather than in this public forum. If you email [registration@vercel.com](mailto:registration@vercel.com), that will create a ticket for the team that can look into this for you. Be sure to include the email, username, and any other relevant additional information about your account and the project you are deploying which help accelerate the security verification process.
